docker/build.sh

```diff
@@ -16,24 +16,24 @@ PROJECT=rawls
 function make_jar()
 {
     echo "building jar..."
-    bash ./docker/run-mysql.sh start
-    bash ./docker/run-opendj.sh start
+#    bash ./docker/run-mysql.sh start
+#    bash ./docker/run-opendj.sh start
```

Comment:
these changes were so i didn't have to wait for jenkins to run the tests when deploying this code to a FiaB